packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.125.0
+------------------------------
+*May 24, 2021*
+
+### Added
+- Reading address from local storage if it exists 
```

Comment:
Is there another pr to populate the address fields coming later?

Reply:
Nope, that's already done so updated the changelog to reflect that